communication/slack-config/channels.yaml

```
@@ -171,6 +171,7 @@ channels:
   - name: hephy-users
     id: C678BMZ89
   - name: hetzner
+  - name: hmc  
```

```suggestion
  - name: hmc-dev
```
Slack channels and names share a namespace and there's already a hmc =/